plugins/koterm.koplugin/terminputtext.lua

```diff
@@ -0,0 +1,391 @@
+--[[
+module used for terminal emulator to override InputText
+]]
+
+local InputText = require("ui/widget/inputtext")
+local UIManager = require("ui/uimanager")
+local dbg = require("dbg")
+local util = require("util")
+
+local esc_seq = {
+    backspace = "\008",
+    cursor_left =  "\027[D",
+    cursor_right = "\027[C",
+    cursor_up =    "\027[A",
+    cursor_down =  "\027[B",
+    cursor_pos1 =  "\027[7~",
+    cursor_end =   "\027[8~",
+    page_up =      "\027[5~",
+    page_down =    "\027[6~",
+}
+
+local TermInputText = InputText:extend{
+    terminal_mode = true,
+    maxr = 40,
+    maxc = 80,
+    strike_callback = nil,
+}
+
+function TermInputText:onTapTextBox(arg, ges)
+    -- disable positioning cursor by tap in emulator mode
+    if self.terminal_mode then
+        return true
+    end
+    return InputText.onTapTextBox(self, arg, ges)
```

Comment:
Can you / do you plan on allowing disabling `terminal_mode` ?  Or was it just a flag to tweak behaviour when all this was in a single InputText?
If it was just a flag, simplify all this: in TermInputText, you are terminal_mode, you'll never need to call the base class equivalent:
```lua
function TermInputText:onTapTextBox(arg, ges)
    -- no positioning cursor by tap in emulator mode
    return true
end
```

Reply:
Yes it is a leftover, I will remove it. It is in there a few times.